mkosi/__init__.py

```diff
@@ -2732,6 +2733,9 @@ def run_verb(args: MkosiArgs, images: Sequence[MkosiConfig]) -> None:
             if args.verb == Verb.qemu:
                 run_qemu(args, last, qemu_device_fds)
 
+            if args.verb == Verb.vmspawn:
+                run_vmspawn(args, last, qemu_device_fds)
```

Comment:
Hmm, should this be a separate verb? In the near term, I guess we want two separate verbs until vmspawn is stable and widely supported. But in the long term, I guess we'd want to have just one verb. So I guess it's best to add a separate verb for now. We might do some renaming later on.